app/views/submissions/index.html.haml

```diff
@@ -36,7 +36,11 @@
             - else
               = submission.editor
           %td= submission.received.strftime("%d-%m-%Y")
-          %td= submission.last_deadline
+          %td    
+            - if submission.last_review != nil && submission.last_review.missed_deadline
```

Comment:
Hmm, przyszła mi tutaj jeszcze jedna kwestia do głowy - mianowicie, metoda `last_review` nie sprawdza stanu recenzji, a `deadline_missed?` tak. Dlatego nie możemy opierać się na tym uproszczeniu, że sprawdzamy tylko stan ostatniej recenzji. Trzeb zrobić to inaczej - w klasie `Submission`:
```ruby
def deadline_missed?
  self.reviews.any?{|r| r.deadline_missed? }
end
```